test/gtest/cba_find2_infer.cpp

```diff
@@ -162,7 +162,11 @@ TEST_P(GPU_ConvBiasActivFind2InferFusionFind_FP32, ConvBiasActivFind2Float_testF
         {miopenTensorActivationY, out_dev.get()},
         {miopenTensorBias, bias_dev.get()},
     };
-
+    if(solutions.empty())
```

Comment:
This is skipping all "unsupported" platforms, not just gfx12, right?

Reply:
In a direct way yes. If `solutions` is empty then the platforms does not support cba.